### PR TITLE
Create the 3.0 release.

### PR DESCRIPTION
**Pull request by abadger**
We're starting to have submissions for new collections for ansible-3.0.
Need to make the 3.0 directory and ansible.in so that those have a place
to land.

### CODE REVIEW

**Comment by felixfontein** [comment]
Shouldn't the directory be called `3` and not `3.0`, since 3.1.0, 3.2.0 etc are also released from it?

**Comment by abadger** [comment]
> Shouldn't the directory be called `3` and not `3.0`, since 3.1.0, 3.2.0 etc are also released from it?

That's a good point.  I agree.  I suppose all of the files inside of the directory with version numbers ought to undergo a similar change in what they're named.  Will need some code in antsibull to do that.

File: 3.0/ansible.in [hunk 1]
```diff
@@ -0,0 +1,78 @@
+amazon.aws
```
Comment (felixfontein) [inline 3.0/ansible.in:1]: The 3.0/ file is still there.

Comment (abadger) [reply 3.0/ansible.in]: Doh!   Fixed.

**Review by felixfontein — APPROVED** [review]

**Pull request merged**